extras/index.js

```diff
@@ -59,17 +59,8 @@ module.exports = generators.Base.extend({
       var patternCollection;
       var destPath = this.options.themePath || './';
 
-      if (this.options.drupalDistro === 'drupal') {
-        if (this.options['drupalDistroVersion-drupal'] === '7.x') {
-          patternCollection = 'drupal-7.x';
-        }
-        if (this.options['drupalDistroVersion-drupal'] === '8.x') {
-          patternCollection = 'drupal-8.x';
-        }
-      } else if (this.options.drupalDistro === 'openatrium') {
-        if (this.options['drupalDistroVersion-openatrium'] === '7.x') {
-          patternCollection = 'openatrium-7.x';
-        }
+      if (['drupal', 'openatrium'].indexOf(this.options.drupalDistro) !== -1) {
```

Comment:
Preserved the distro validation in case other generators add distros faster than PLS.